### DISCUSSION TITLE
Time limited access to photo/album

### DISCUSSION THREAD

**Discussion by lxs** [tripupapp/tripup-server#4] [Ideas]
**Gist:** Creator can impose time limited access to their content.

**Summary:** Time constrained access to endpoint by creator imposing time limit as to when URL can be used.

Examples:
1. "available only after 14:30 BST today"
2. "no longer available after 21:00 PST tomorrow"

The first example is a type of "information embargo", such as publication of financial data or some press release. This could be meaningful to artists/photographers who wish to delay the availability of their digital content to match with some real-world event.

**Comment by vin047** [comment]
Although an interesting app idea on its own, I question the use case for an encrypted photo service having such a feature, for similar reasons mentioned in https://github.com/tripupapp/tripup-server/discussions/7#discussioncomment-842454.
